CSS/HomePg.css

```diff
@@ -0,0 +1,69 @@
+.headline {
+  background-color: #1c1a19;
+  margin: 0%;
+  background-image: url(/images/bdimage.svg);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  height: auto;
+  padding: 120px 24px;
+}
+
+.mainText {
+  margin: 0%;
+  padding-top: 30px;
+  color: #ff6b00;
+}
+
+.textcolor {
+  color: #fff;
+}
+
+.contactIcons {
+  list-style: none;
+  display: flex;
+  justify-content: space-evenly;
+  padding: 0%;
+}
+
+@media screen and (min-width: 768px) {
+  .headline {
+    background-image: url(/images/D_HomePgBG.svg);
+    background-repeat: no-repeat;
+    background-position: right;
+    background-size: 500.13px 769.27px;
+    display: grid;
+    grid-template-areas:
+      ". A A A"
+      ". A A A"
+      "D A A A"
+      "D A A A"
+      "D B B B"
+      "D C C ."
+      "D C C ."
+      "D C C .";
+    column-gap: 20px;
+  }
```

Comment:
- [ ] Kindly here you can give the headline section `height: 400px` to make it take a bigger width like the Figma design.

Your design
![image](https://user-images.githubusercontent.com/35535130/166123194-20602cda-c5f6-4673-8609-731c8900b6a8.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/166123207-e51c1943-be07-4387-8789-a602122dbd02.png)